src/main/resources/templates/fragments/accountSummaryTable.html

```diff
@@ -28,16 +28,16 @@
             <td class="table-border-right"></td>
             <td th:text="'€' + ${data.investments}">€1000.00</td>
         </tr>
-        <tr>
-            <td th:text="${data.translations.marketplaceBought}" class="text-left table-border-right">Marketplace shares bought</td>
-            <td class="table-border-right"></td>
-            <td th:text="'€' + ${data.sharesBought}">€371.00</td>
-        </tr>
-        <tr class="table-grey-background table-border-bottom">
-            <td th:text="${data.translations.marketplaceSold}" class="text-left table-border-right table-no-bottom-border">Marketplace shares sold</td>
-            <td th:text="'€' + ${data.sharesSold}" class="table-border-right table-no-bottom-border">€3,571.00</td>
-            <td class="table-no-bottom-border"></td>
-        </tr>
+<!--        <tr>-->
+<!--            <td th:text="${data.translations.marketplaceBought}" class="text-left table-border-right">Marketplace shares bought</td>-->
+<!--            <td class="table-border-right"></td>-->
+<!--            <td th:text="'€' + ${data.sharesBought}">€371.00</td>-->
+<!--        </tr>-->
+<!--        <tr class="table-grey-background table-border-bottom">-->
+<!--            <td th:text="${data.translations.marketplaceSold}" class="text-left table-border-right table-no-bottom-border">Marketplace shares sold</td>-->
+<!--            <td th:text="'€' + ${data.sharesSold}" class="table-border-right table-no-bottom-border">€3,571.00</td>-->
+<!--            <td class="table-no-bottom-border"></td>-->
+<!--        </tr>-->
```

Comment:
Remove this code and also from TransactionsSummary `sharesBought` and `sharesSold`. Everything will be added when the final information is provided by CBDO.